src/palace/manager/api/overdrive.py

```diff
@@ -1674,18 +1744,20 @@ def release_hold(self, patron, pin, licensepool):
         )
         response = self.patron_request(patron, pin, url, method="DELETE")
         if response.status_code // 100 == 2 or response.status_code == 404:
-            return True
+            return
         if not response.content:
             raise CannotReleaseHold()
         data = response.json()
         if not "errorCode" in data:
             raise CannotReleaseHold()
         if data["errorCode"] == "PatronDoesntHaveTitleOnHold":
             # There was never a hold to begin with, so we're fine.
-            return True
+            return
```

Comment:
Base class only returned None, so I removed the bool return values here that were never used anyway.